PVSSpringBoot/src/main/java/com/example/PVSSpringBoot/Entities/Product.java

```diff
@@ -0,0 +1,52 @@
+package com.example.PVSSpringBoot.Entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Objects;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "product")
+@RequiredArgsConstructor
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name = "product_name", nullable = false)
+    @NonNull
+    private String productName;
+
+    @Column(name = "brand_name", nullable = false)
+    @NonNull
+    private String brandName;
```

Comment:
What do you mean by brand name?

Reply:
a brand that can have multiple products with different names and uses (ex:Friskies producces both cat food and dog food) + was in the ERD